src/test/app/Vault_test.cpp

```diff
@@ -4795,6 +4831,136 @@ class Vault_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testDelegate()
+    {
+        using namespace test::jtx;
+
+        Env env(*this, testable_amendments());
+        Account alice{"alice"};
+        Account bob{"bob"};
+        Account carol{"carol"};
+
+        struct CaseArgs
+        {
+            PrettyAsset asset = xrpIssue();
+        };
+
+        auto const xrpBalance = [](Env const& env,
+                                   Account const& account) -> long {
+            auto sle = env.le(keylet::account(account.id()));
+            assert(sle != nullptr);
+            return sle->getFieldAmount(sfBalance).xrp().drops();
```

Comment:
This `assert` could be a `BEAST_EXPECT`.
```
if(!BEAST_EXPECT(sle != nullptr))
    return 0; // or some weird value like -1
```
Don't bother checking the result, because the test has failed, so it'll cascade.

Reply:
Thanks, done in 328369b57b155039b3da96842ee9c6203ca1959d